mailbox/client_conn.go

```diff
@@ -147,12 +133,18 @@ func NewClientConn(ctx context.Context, sid [64]byte, serverHost string,
 
 	receiveSID := GetSID(sid, true)
 	sendSID := GetSID(sid, false)
+	mailBoxInfo := &mailboxInfo{
+		addr:    serverHost,
```

Comment:
Do we need the `devServer` boolean here too which ignores self-signed certs? Or don't we currently support that on the client side?

Reply:
(same as for `ServerConn`)